packages/payment-backend/src/service.spec.ts

```diff
@@ -268,3 +285,51 @@ describe('Test paidPaymentExists', () => {
     await expect(paidPaymentExists(params)).rejects.toThrow()
   })
 })
+
+describe('Test save payment filters for order', () => {
+  it('Should throw error with no backend url set', async () => {
+    process.env.PAYMENT_BACKEND_URL = ''
+    await expect(savePaymentFiltersAdmin(paymentFiltersMock)).rejects.toThrow(
+      'No payment API backend URL set'
+    )
+  })
+
+  it('Should save payment filters for order', async () => {
+    process.env.PAYMENT_BACKEND_URL = 'test.dev.hel'
+
+    const paymentFilterReqeustDataMock = [
+      {
+        filterId: 'faa1a6f9-0f01-4d54-89bb-6ff2e7314124',
+        createdAt: '1619157868',
+        namespace: 'testNameSpace',
+        referenceId: '145d8829-07b7-4b03-ab0e-24063958ab9b',
+        referenceType: 'order',
+        type: 'order',
```

Comment:
What is this type ? Should this be removed

Reply:
Yes, I'll remove it.

Reply:
This is filter type. I can rename it to filterType.